lib/model.js

```diff
@@ -140,6 +133,9 @@ var Events = require('./events'),
                     return model.ready().then(function () {
                         return model.storage.insert(model);
```

Comment:
Do we realy able to store model without an id attribute? Maybe we should throw an error before calling the `model.storage.insert` or just at the beginning of the save method?